mitmproxy/proxy/protocol/tls.py

```diff
@@ -465,6 +465,8 @@ def _establish_tls_with_client(self):
         self.log("Establish TLS with client", "debug")
         cert, key, chain_file = self._find_cert()
 
+        self.client_conn.mitmcert = cert
```

Comment:
Can we move this [here](https://github.com/mitmproxy/mitmproxy/blob/4cec88fc7fcf8a5c16c0639961c27792cd73af13/mitmproxy/connections.py#L113)?